op-batcher/batcher/channel_manager_test.go

```diff
@@ -19,11 +18,97 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TestChannelManagerAddL2Block tests adding an L2 block to the channel manager.
+func TestChannelManagerAddL2Block(t *testing.T) {
```

Comment:
Is the primary addition of this test checking that `m.blocks` is as expected? Otherwise it looks like the reorg tests.